cmd/strelaypoolsrv/main.go

```diff
@@ -335,6 +345,7 @@ func handleRequest(w http.ResponseWriter, r *http.Request) {
 	timer := prometheus.NewTimer(apiRequestsSeconds.WithLabelValues(r.Method))
 
 	lw := NewLoggingResponseWriter(w)
+	w = lw
```

Comment:
This is weird? Get rid of one of them?

Reply:
We can't get rid of `w` as it's a parameter and will always be in scope, so the risk is that we will use it anyway. We can get rid of `lw` and instead use a type assertion in the defer, do you think that's cleaner?

Reply:
Fixed-up to do that, it was cleaner I think.